README.md

```diff
@@ -1 +1,90 @@
-# furniture
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# FirstCapStone
+
+> This project is My First Capstone, I made it using HTML, CSS and JavaScript only. 
+
+
+
+## Built With
+- HTML
+- CSS
+- JavaScript
+
+
+## Live Demo
+
+- [Live link](https://ahsan12356.github.io/furniture/)
```

Comment:
- [ ] Good job deploying the site to GitHub pages, however, your background images are not displaying in the deployed version. they are working all right in the local version so the issue is only with the GitHub page deployment, make sure the links to the images are correct in order to be displayed properly on the live website

![Screenshot_20221028_221553](https://user-images.githubusercontent.com/25227022/198734562-002e2b5d-b517-484f-b422-5ac318682511.png)

Reply:
Can you please elaborate that how can I resolve that issue? Because I have given the links correctly that is why the images are being shown locally in visual studio.

Reply:
@Ahsan12356, sorry for the late response. did you figure it out or do you need some assistance?